SPECS/buildah/buildah.spec

```
@@ -0,0 +1,1973 @@
+%global with_bundled 1
+
+
+%global with_debug 1
+
+
+%if 0%{?with_debug}
+%global _find_debuginfo_dwz_opts %{nil}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package   %{nil}
+%endif
+
+%global provider github
+%global provider_tld com
+%global project containers
+%global repo buildah
+# https://github.com/containers/buildah
+%global import_path %{provider}.%{provider_tld}/%{project}/%{repo}
+%global git0 https://%{import_path}
+
+# Used for comparing with latest upstream tag
+# to decide whether to autobuild (non-rawhide only)
+%define built_tag v1.18.0
+%define built_tag_strip %(b=%{built_tag}; echo ${b:1})
+%define download_url https://%{import_path}/archive/%{built_tag}.tar.gz
+
+Name:         %{repo}
+Version:      1.18.0
+Release:      6%{?dist}
+Summary:      A command line tool used for creating OCI Images
+License:      Apache-2.0
+Vendor:       Microsoft Corporation
+Distribution: Mariner
+URL: https://%{name}.io
+Source: %{download_url}#/%{name}-%{version}.tar.gz
```

It seems `download_url` is used in only one place, so we can simplify a bit:
```suggestion

Name:         %{repo}
Version:      1.18.0
Release:      6%{?dist}
Summary:      A command line tool used for creating OCI Images
License:      Apache-2.0
Vendor:       Microsoft Corporation
Distribution: Mariner
URL: https://%{name}.io
Source: https://%{import_path}/archive/%{built_tag}.tar.gz#/%{name}-%{version}.tar.gz
```

---

Done!